### PR TITLE
Contacts support

### PR DESCRIPTION
**Pull request by marcos-albornoz**
Added support to set contacts

### CODE REVIEW

**Review by ricrodriguezg — APPROVED** [review]

File: pkg/venafi/cloud/connector_test.go [hunk 1]
```diff
@@ -832,7 +832,7 @@ func TestSetPolicy(t *testing.T) {
 	//now update policy.
 	t.Log("updating policy, modifying ps.Policy.Subject.OrgUnits = []string{\"DevOps\", \"QA\"}.")
 	localPolicy.Policy.Subject.OrgUnits = []string{"DevOps", "QA"}
-	policyName = appName + "\\" + test.RandCitName()
+	//policyName = appName + "\\" + test.RandCitName()
```
Comment (luispresuelVenafi) [inline pkg/venafi/cloud/connector_test.go:835]: Can we remove commented code?

**Review by luispresuelVenafi — APPROVED** [review]
Changes looks good

File: pkg/venafi/tpp/connector.go [hunk 2]
```diff
@@ -1001,6 +1043,125 @@ func (c *Connector) SetPolicy(name string, ps *policy.PolicySpecification) (stri
...
+		return nil, err
+	}
+	browseIdentitiesResponse, err := parseBrowseIdentitiesResult(statusCode, status, body)
+	if err != nil {
+		return nil, err
+	}
+	return &browseIdentitiesResponse, nil
+}
+
+func (c *Connector) identitySelf() (*policy.IdentitySelfResponse, error) {
```
Comment (rvelaVenafi) [inline pkg/venafi/tpp/connector.go:1152]: Maybe this function should be called selfIdentity instead of identitySelf, makes more sense

**Review by rvelaVenafi — CHANGES_REQUESTED** [review]

File: pkg/venafi/tpp/connector.go (hunk 2 above)
Comment (marcos-albornoz) [reply pkg/venafi/tpp/connector.go:1152]: got it... doing the change

**Review by rvelaVenafi — APPROVED** [review]

**Pull request merged**